source/geometries/Next100FieldCage.cc

```diff
@@ -274,7 +283,7 @@ void Next100FieldCage::DefineMaterials()
   steel_ = materials::Steel316Ti();
   // In Geant4 11.0.0, a bug in treating the OpBoundaryProcess produced in the surface makes the code fail.
   // This is avoided by setting an empty G4MaterialPropertiesTable of the G4Material.
-  steel_->SetMaterialPropertiesTable(new G4MaterialPropertiesTable());
+  steel_->SetMaterialPropertiesTable(opticalprops::Steel());
```

Comment:
I'm not sure this is correct. To simulate a reflection on a material, you have to define an optical surface wrapped around it with the desired properties, while here you give the properties to the material itself.

Reply:
You are right, I don't know why I did it like this. I'll add a fix for this in my next commits and make sure it works with a test.